### ISSUE TITLE
Document problem on yolo v3 training scripts and logs?

### ISSUE THREAD

**Issue by wzhang1** [dmlc/gluon-cv#1059]
This is a purely doc issue

The current doc on yolo v3 training scripts and logs
https://gluon-cv.mxnet.io/model_zoo/detection.html#id63

seems have some problem,
1, 

yolo3_darknet53_coco 3 (320x320) | 33.6/54.1/35.8 | shell script | log
-- | -- | -- | --
yolo3_darknet53_coco 3 (416x416) | 36.0/57.2/38.7 | shell script | log
yolo3_darknet53_coco 3 (608x608) | 37.0/58.2/40.1 | shell script | log

all three experiments on different resolutions seems to be the same 416x416

2, after epoch 267, the log went back to epoch 250, is there anything  special on this

**Comment by EletronicElephant** [comment]
Hello, I'm also working on these code these days and very glad to help you.

For Question 1, as this implementation supports multi-scale training, the resolution might stands for the resolution when testing.

For Q2, maybe they found that the test mAP starts to drop after the learning rate was decreased to 1e-5 at epoch 250, so they restart the training from the params which has the best test mAP.
